tests/conftest.py

```diff
@@ -7,7 +7,7 @@
 # You can configure optional parameters by overriding the value of ragger.configuration.OPTIONAL_CONFIGURATION
 # Please refer to ragger/conftest/configuration.py for their descriptions and accepted values
 
-configuration.OPTIONAL.APP_DIR = "tests/ethereum_build/"
+configuration.OPTIONAL.APP_DIR = "tests/.test_dependencies/"
```

Comment:
Should we remove the "." here ?

Reply:
In `.test_dependencies` ? Probably not, it's ok if it's a hidden folder, this is not something the user will manipulate often.